ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -173,6 +174,33 @@ interface {{ vlan_interface }}
    vrrp {{ vlan_interfaces[vlan_interface].vrrp.virtual_router }} ipv6 {{ vlan_interfaces[vlan_interface].vrrp.ipv6 }}
 {%         endif %}
 {%     endif %}
+{# New improved "vrrp" data model to support multiple IDs #}
+{%     if vlan_interfaces[vlan_interface].vrrp_ids is arista.avd.defined %}
+{%         for vrid in vlan_interfaces[vlan_interface].vrrp_ids | arista.avd.natural_sort %}
+{%             if vrid.priority_level is arista.avd.defined %}
+   vrrp {{ vrid.id }} priority-level {{ vrid.priority_level }}
+{%             endif %}
+{%             if vrid.advertisement.interval is arista.avd.defined %}
+   vrrp {{ vrid.id }} advertisement interval {{ vrid.advertisement.interval }}
+{%             endif %}
+{%             if vrid.preempt.enabled is arista.avd.defined(true) %}
+{%                 set delay_cli = 'vrrp ' ~ vrid.id ~ ' preempt delay' %}
```

Comment:
"delay" should be only printed if minimum of reload is defined.
Right now the configuration generated is wrong:
```yaml
    vrrp_ids:
    - id: 60
      priority_level: 100
      advertisement:
        interval: 30
      preempt:
        enabled: true
      ipv4:
        address: 10.0.60.1
```
This generates the following command:
```eos
   vrrp 60 priority-level 100
   vrrp 60 advertisement interval 30
   vrrp 60 preempt delay  <-- here
   vrrp 60 ipv4 10.0.60.1
```
Which is not a valid configuration:
```eos
POD1_LEAF01A(config)#int vlan 10
POD1_LEAF01A(config-if-Vl10)#vrrp 10 priority-level 100
POD1_LEAF01A(config-if-Vl10)#vrrp 10 preempt ?
  delay  Required if either the minimum or reload keywords are specified
  <cr>

POD1_LEAF01A(config-if-Vl10)#vrrp 10 preempt
POD1_LEAF01A(config-if-Vl10)#sh ac
interface Vlan10
POD1_LEAF01A(config-if-Vl10)#vrrp 10 preempt ?
  delay  Required if either the minimum or reload keywords are specified
  <cr>

POD1_LEAF01A(config-if-Vl10)#vrrp 10 preempt delay ?
  minimum  Specify the minimum time in seconds for the local router to wait
           before taking over the active role. Default is 0 seconds.
  reload   Specify the time after a reload in seconds for the local router to
           wait before taking over the active role. Default is 0 seconds

POD1_LEAF01A(config-if-Vl10)#vrrp 10 preempt delay
% Incomplete command
POD1_LEAF01A(config-if-Vl10)#
```

Reply:
Fixed - also testing this case via molecule.